sites-enabled/10-www.freecodecamp.org.conf

```diff
@@ -1055,5 +1064,9 @@ server {
 
   server_name .freecodecamp.org;
 
+  if ($euler_redirects_uri) {
+    return 301 https://www.freecodecamp.org$euler_redirects_uri;
+  }
+
```

Comment:
Putting aside the above discussion, these do not belong in this directive. It has a clear responsibility (http -> https), and the euler redirects are (or should be) separate from that. The same goes for the fcc -> www.fcc directive.

Regardless of where we put the maps, I would put the redirection in the same place as the legacy redirection (i.e. in the main server directive, after `include snippets/common/legacy-redirects.conf;`